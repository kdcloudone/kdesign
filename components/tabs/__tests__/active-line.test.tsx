import React from 'react'
import { render, mount } from 'enzyme'
import mountTest from '../../../tests/shared/mountTest'
import Line from '../active-line'

describe('Tabs Active Line', () => {
  mountTest(Line)

  it('renders correctly', () => {
    const wrapper = render(<Line left={0} width={100} />)
    expect(wrapper).toMatchSnapshot()
  })

  it('should have displayName static property', () => {
    const wrapper = mount(<Line left={0} width={100} />)
    expect((wrapper.type() as any).displayName).toBe('Line')
  })
})
