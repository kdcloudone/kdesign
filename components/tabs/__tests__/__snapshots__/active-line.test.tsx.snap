// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs Active Line renders correctly 1`] = `
<span
  class="kd-active-line"
  style="left:0px;width:100px"
/>
`;
